examples/js/animation/MMDAnimationHelper.js

```diff
@@ -660,19 +679,28 @@ THREE.MMDAnimationHelper = ( function () {
 				grantResultMap.set( boneIndex, quaternion.copy( bone.quaternion ) );
 
 				// @TODO: Support global grant and grant position
-				if ( grantSolver && boneData.grant &&
-					! boneData.grant.isLocal && boneData.grant.affectRotation ) {
+				if ( grantSolver && boneData.grant && ! boneData.grant.isLocal ) {
 
 					var parentIndex = boneData.grant.parentIndex;
 					var ratio = boneData.grant.ratio;
 
-					if ( ! grantResultMap.has( parentIndex ) ) {
-
-						updateOne( mesh, parentIndex, ikSolver, grantSolver );
+					if( boneData.grant.affectRotation ) {
+	
+						if ( ! grantResultMap.has( parentIndex ) ) {
+	
+							updateOne( mesh, parentIndex, ikSolver, grantSolver );
+	
+						}
+	
+						grantSolver.addGrantRotation( bone, grantResultMap.get( parentIndex ), ratio );
 
 					}
 
-					grantSolver.addGrantRotation( bone, grantResultMap.get( parentIndex ), ratio );
+					if( boneData.grant.affectPosition && bones[parentIndex].userData.MMD ) {
+
+						grantSolver.addGrantPosition( bone, bones[parentIndex], ratio );
```

Comment:
According to the PMX spec, grant position seems to need to use `grantResultMap`, too.